src/Components/CryptoCurrencyNetworks.res

```diff
@@ -0,0 +1,44 @@
+@react.component
+let make = () => {
+  open DropdownField
+  let currencyVal = Recoil.useRecoilValueFromAtom(RecoilAtoms.userCurrency)
+  let {config, localeString} = Recoil.useRecoilValueFromAtom(RecoilAtoms.configAtom)
+  let (cryptoCurrencyNetworks, setCryptoCurrencyNetworks) = Recoil.useRecoilState(
+    RecoilAtoms.cryptoCurrencyNetworks,
+  )
+
+  let dropdownOptions =
+    Utils.currencyNetworksDict
+    ->Dict.get(currencyVal)
+    ->Option.getOr([])
+    ->Array.map(item => {
+      label: item
+      ->String.toUpperCase
+      ->String.split("_")
```

Comment:
Please move this function to Utils next to the other case transformation functions

Reply:
Updated